docs/reference/library.md

```diff
@@ -167,7 +167,7 @@ Then feeding its result into [mkCabalProjectPkgSet](#mkcabalprojectpkgset) passi
 | `shellFor`        | Function                                         | [`shellFor`](#shellfor)                                                     |
 | `ghcWithHoogle`   | Function                                         | [`ghcWithHoogle`](#ghcwithhoogle)                                           | 
 | `ghcWithPackages` | Function                                         | [`ghcWithPackages`](#ghcwithpackages)                                       |
-
+| `cross`           | Attrset                                          | Like `pkgs.pkgsCross.X` from nixpkgs `cross.X` returns the project results for cross compilation to X.  So `cross.ghcjs.hsPkgs` is the same as `hsPkgs` but compiled with ghcjs |
```

Comment:
Should we call it `hsPkgsCross`?

Reply:
From https://nixos.wiki/wiki/Cross_Compiling:

> The basic idea is to use `pkgsCross.platform` instead of `pkgs`

Since we are doing this for a project (not a package set) I have renamed it `projectCross`.

Also because of that wording I have renamed the `shellFor` arg `crossPlatforms`.